### PR TITLE
Amls 5673 - Money laundering regulation content update

### PR DESCRIPTION
**Pull request by matthewharris1**
Removal of anti-money-laundering supervision text replaced with suitable content.

## How Has This Been Tested?
Unit
Manual

## Checklist

- [ ] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala [hunk 1]
```diff
@@ -46,20 +50,21 @@ class ExpectedAMLSTurnoverController @Inject() (val dataCacheConnector: DataCach
             (for {
               businessActivities <- cache.getEntry[BusinessActivities](BusinessActivities.key)
               expectedTurnover <- businessActivities.expectedAMLSTurnover
-            } yield Ok(expected_amls_turnover(Form2[ExpectedAMLSTurnover](expectedTurnover), edit, businessMatching)))
-              .getOrElse (Ok(expected_amls_turnover(EmptyForm, edit, businessMatching)))
-          }) getOrElse Ok(expected_amls_turnover(EmptyForm, edit, None))
+            } yield Ok(expected_amls_turnover(Form2[ExpectedAMLSTurnover](expectedTurnover), edit, businessMatching, businessMatching.alphabeticalBusinessActivitiesLowerCase())))
```
Comment (ajchilts) [inline app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala:53]: Maybe just a little formatting to wrap over multiple lines so as more readable?

File: test/views/businessactivities/expected_amls_turnoverSpec.scala [hunk 2]
```diff
@@ -60,7 +59,7 @@ class expected_amls_turnoverSpec extends AmlsViewSpec with MustMatchers  {
 
       val form2: ValidForm[ExpectedAMLSTurnover] = Form2(ExpectedAMLSTurnover.Third)
 
-      def view = views.html.businessactivities.expected_amls_turnover(form2, true, businessMatching2)
+      def view = views.html.businessactivities.expected_amls_turnover(form2, true, None, Some(List("accountancy service provider", "estate agency business")))
```
Comment (ajchilts) [inline test/views/businessactivities/expected_amls_turnoverSpec.scala:62]: Any reason you are replacing businessMatching2 with None?

File: test/views/businessactivities/expected_amls_turnoverSpec.scala [hunk 3]
```diff
@@ -48,7 +47,7 @@ class expected_amls_turnoverSpec extends AmlsViewSpec with MustMatchers  {
 
       val form2: ValidForm[ExpectedAMLSTurnover] = Form2(ExpectedAMLSTurnover.Third)
 
-      def view = views.html.businessactivities.expected_amls_turnover(form2, true, businessMatching)
+      def view = views.html.businessactivities.expected_amls_turnover(form2, true, None, Some(List("accountancy service provider")))
```
Comment (ajchilts) [inline test/views/businessactivities/expected_amls_turnoverSpec.scala:50]: Any reason you are replacing businessMatching with None?

File: test/views/businessactivities/expected_amls_turnoverSpec.scala (hunk 2 above)
Comment (matthewharris1) [reply test/views/businessactivities/expected_amls_turnoverSpec.scala:62]: I'm not testing businessMatching2 anymore I'm testing on the optional list I have provided

File: test/views/businessactivities/expected_amls_turnoverSpec.scala (hunk 3 above)
Comment (matthewharris1) [reply test/views/businessactivities/expected_amls_turnoverSpec.scala:50]: ^

File: app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala (hunk 1 above)
Comment (matthewharris1) [reply app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala:53]: can do

Comment (matthewharris1) [reply app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala:53]: In all honestly I think this is as good as the formatting can get without making it really ugly.
Screenshot 2020-04-01 at 14 49 44

Comment (ajchilts) [reply app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala:53]: Screenshot 2020-04-01 at 14 58 34

Comment (ajchilts) [reply app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala:53]: Its not a show stopper. I tend to wrap lines over a certain length. ;-)

Comment (matthewharris1) [reply app/controllers/businessactivities/ExpectedAMLSTurnoverController.scala]: Guess its personal preference but ive refactored

**Pull request closed without merging**